pantab/tests/conftest.py

```diff
@@ -59,15 +57,15 @@ def df():
                 0,
                 0,
                 0,
+                # np.nan,
```

Comment:
These are commented out because of a bug upstream in pandas. see https://github.com/pandas-dev/pandas/issues/56777